controller/address_controller.go

```diff
@@ -0,0 +1,50 @@
+package controller
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/Marcxz/academy-go-q32021/usecase"
+)
+
+var (
```

Comment:
If you only have one global variable, you don't need a block statement, do it in one line:
```go
var au = usecase.NewAddressUseCase()
```